dbms/src/Columns/ColumnAggregateFunction.h

```diff
@@ -165,10 +165,27 @@ class ColumnAggregateFunction final : public COWPtrHelper<IColumn, ColumnAggrega
 
     const char * deserializeAndInsertFromArena(const char * src_arena, const TiDB::TiDBCollatorPtr &) override;
 
+    void countSerializeByteSizeUnique(
```

Comment:
Would it make more sense to rename `countSerializeByteSizeUnique` to `countSerializeUniqueByteSize` for better readability? 
Similarly, the name for other methods could follow the same pattern. E.g. `serializeToPosUnique` to `serializeUniqueToPos` and `deserializeAndInsertFromPosUnique` to `deserializeUniqueAndInsertFromPos`.